docs/conf/portland/2021/news/announcing-schedule.rst

```diff
@@ -0,0 +1,61 @@
+:template: {{year}}/generic.html
+
+.. post:: Mar 17, 2020
+   :tags: {{year}}, portland-{{year}}, schedule
+
+Announcing Full Schedule
+========================
+
+Hey everyone!
+
+We're just over a month away from the online Portland conference, and we're working hard to make sure it'll be as welcoming and as fun as the in-person event you're probably familar with.
+
+Today we're announcing the full presentation schedule and the platform we'll be hosting the conference on.
+We are working on last-minute details for a few more activities during the conference, so look out for more information in our next mailing.
+
+Conference platform
+--------------------
+
+Following the success of last year's virtual format, the conference will again be held on https://hopin.to/. Hopin not only provides a stage for our amazing speakers, but also has integrated chat, unconference rooms with breakout calls, and expo booths. We're updating our Welcome Wagon guide, to be released soon, so it'll all feel familiar before you've even arrived.
+
+Full presentation schedule
+--------------------------
+
+The conference website is now updated with the `talk schedule <https://www.writethedocs.org/conf/portland/{{year}}/schedule/>`_ for the main stage presentations, so you can start planning your time (and getting excited!) for the conference. We're excited to continue the new tradition of live Q&A after most talks, which will come from your questions but be moderated by our MC.
+
+One of the big changes in the schedule based on the community feedback is scheduling more breaks during the day. We are all trying to balance our screen time, so we wanted to make sure that your eyes get their rest! We hope that this adjustment will help you stay alert and happy throughout the conferences.
+
+Another change that we made is to move the social event to an earlier time and shorten the activity time slots. We want you to have fun socializing with your fellow documentarians, but we understand that your family and home time is also important these days, so hopefully this will help us stay engaged and enjoy the social activities even more.
```

Comment:
```suggestion
Another change that we made is to move the social event to an earlier time. We want you to have fun socializing with your fellow documentarians, but we understand that your family and home time is also important these days, so hopefully this will help us stay engaged and enjoy the social activities even more.
```